README.md

```diff
@@ -105,6 +105,8 @@ func main() {
 }
 ```
 
+> Note: use `verifier.DisableCatchAllCheck()` to disable catchAll check
```

Comment:
It would be more appropriate to put this comment on line 112， and remove the "Note"